README.md

```diff
@@ -57,15 +57,17 @@ The schedule is fairly static:
 
 Time (PDT - **UTC/Z** - CET) | Topic | +Hangout
 :------------------------: | :---: | :------:
-11:30PDT **18:30Z** 19:30CET | Apps on IPFS, electron | [Video Room](https://plus.google.com/hangouts/_/grdn26fpdroghn5wa56mhpxz34a)
-12:00PDT **19:00Z** 20:00CET | infrastructure | [Video Room](https://plus.google.com/hangouts/_/g6irrqkylecjoo2k7e7wzkkkgua)
-12:30PDT **19:30Z** 20:30CET | node-ipfs | [Video Room](https://plus.google.com/hangouts/_/gyafa4mpgz7g6jntijoxshfe3ma)
-13:00PDT **20:00Z** 21:00CET | go-ipfs | [Video Room](https://plus.google.com/hangouts/_/g4hc3dnpdvwsklyfd2sxhkwbgqa)
-13:30PDT **20:30Z** 21:30CET | testing + ci | [Video Room](https://plus.google.com/hangouts/_/gwn656w2cihn7lekdarfzhaquea)
-14:00PDT **21:00Z** 22:00CET | protocol + specs | [Video Room](https://plus.google.com/hangouts/_/gxvjk6v6xrc64hcs44phm4c2qaa)
-14:30PDT **21:30Z** 22:30CET | bitswap ml | [Video Room](https://plus.google.com/hangouts/_/grcpjefkp4fv4zqz3xe4ty3mbea)
-
-- Data Structures -- [Video Room](https://plus.google.com/hangouts/_/g7slan3ecrylra7robofp53p6ia)
+11:30PDT **18:00Z** 19:30CET | Apps on IPFS | [Video Room](https://plus.google.com/hangouts/_/grdn26fpdroghn5wa56mhpxz34a)
```

Comment:
Not sure what you mean by 'stale'? Every single one of them works for me and I've added them to the community calendar events, joined by a calendar alarm with the hopes of people getting the alarm (for those who have subscribed the cal) and jump directly into the hangout.